### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by SaturnusDJ**

### CODE REVIEW

**Comment by stuwil** [comment]
Thanks @SaturnusDJ! Can you make the following change in tinder-desktop.js please:

```
.registerAvailableLanguageKeys(['en', 'nl'], {
  'en-*': 'en',
  'nl-*': 'nl'
})
```

File: desktop-app/js/tinder-desktop.js [hunk 1]
```diff
@@ -20,8 +20,8 @@
         prefix: 'locales/',
         suffix: '.json'
       })
-      .registerAvailableLanguageKeys(['en'], {
-        'en-*': 'en'
+      .registerAvailableLanguageKeys(['en', 'nl'], {
+        'en-*': 'en',
       })
```
Comment (stuwil) [inline desktop-app/js/tinder-desktop.js:26]: Needs the second string mapping

**Comment by SaturnusDJ** [comment]
That's weird, it's still in the commit queue. Well, on its way now.

**Comment by stuwil** [comment]
Thanks @SaturnusDJ!

**Pull request merged**